CRM/Utils/System/Base.php

```diff
@@ -945,4 +945,21 @@ public function synchronizeUsers() {
     return [];
   }
 
+  /**
+   * Send an HTTP Response base on PSR HTTP RespnseInterface repsonse.
+   * @param \Psr\Http\Message\ResponseInterface $response
+   */
+  public function sendResponse(\Psr\Http\Message\ResponseInterface $response) {
+    if (function_exists('http_response_code')) {
+      // PHP 5.4+
+      http_response_code($response->getStatusCode());
+    }
+    else {
+      header('X-PHP-Response-Code: ' . $response->getStatusCode(), TRUE, $statusCode->getStatusCode());
```

Comment:
Can we at least have a comment why we need this - if we are keeping support for php < 5.4 to reduce the riskiness of the PR can we flag that?

Reply:
Yeah, I think it'd be fine to just use `http_response_code()`  and remove the conditional.

However, that's a minor cleanup - doesn't need to block this.